app/views/support/providers/new_provider.html.erb

```diff
@@ -0,0 +1,57 @@
+<%= render PageTitle.new(title: "support.providers.new_provider") %>
```

Comment:
Think we could just do:

```suggestion
<%= render PageTitle.new(title: ".new_provider") %>
```

Wouldn't it point to the same path?

Reply:
Nope it points to 

`en.components.page_titles.new_provider`